test/integration/serial_test.rb

```diff
@@ -0,0 +1,22 @@
+class SerialTest < ActionDispatch::IntegrationTest
+  setup do
+    site = create(:site)
+    3.times do
+      create(:serial, :with_posts, site: site)
+    end
+
+    switch_domain(site.fqdn)
+  end
+
+  test "/serials" do
+    visit "/serials"
+
+    serial_titles = find_all "ul li p.serial-summary__title"
```

Comment:
We need more discussion about our coding style, but I don't care about that on this PR.